benchmarks/auto_tune/auto_tune.sh

```diff
@@ -9,6 +9,7 @@ VLLM_LOGGING_LEVEL=${VLLM_LOGGING_LEVEL:-INFO}
 BASE=${BASE:-"$SCRIPT_DIR/../../.."}
 MODEL=${MODEL:-"meta-llama/Llama-3.1-8B-Instruct"}
 SYSTEM=${SYSTEM:-"TPU"}
+MODEL_IMPL_TYPE=${MODEL_IMPL_TYPE:-"vllm"}
```

Comment:
I don't think this env var belongs to vLLM, should we add it here?

Reply:
You're right - I just didn't know what a better location is. I can remove it from here for now.